.expeditor/verify.pipeline.yml

```diff
@@ -587,6 +587,34 @@ steps:
       - gopath-checkout#v1.0.1:
           import: github.com/chef/automate
 
+  - label: "compliance-service scanner with inspec master"
+    command:
```

Comment:
add `soft_fail: true` ?

Reply:
For now, totally... but at some point, once trust is built up and the system is stabilized, I'd like that removed and eyeballs to hit it when it goes red.